src/Explorer/Explorer.tsx

```diff
@@ -2548,4 +2554,18 @@ export default class Explorer {
           />
         );
   }
+
+  public openDeleteDatabaseConfirmationPane(): void {
+    // enableKoPanel which allows the users to switch back to using the old knockout panels.
+    this.isFeatureEnabled(Constants.Features.enableKOPanel)
```

Comment:
Remove this and delete any code from the old panel. I think we need to be aggressive in our migration strategy. We can always revert the commit if there is a major problem

Reply:
Okay Sure. Removed